timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/rdf/CollectionDescription.java

```diff
@@ -20,6 +20,7 @@ private CollectionDescription(String entityTypeName, String vreName, String rdfU
     this.entityTypeName = entityTypeName;
     this.vreName = vreName;
     this.rdfUri = rdfUri;
+    this.unknown = entityTypeName != null && entityTypeName.endsWith("unknown");
```

Comment:
Also, this is only needed if an unknown collection is ever created using createCollectionDescription instead of getDefault() does that ever happen?